smithy-swift-codegen/src/test/kotlin/StructEncodeGenerationTests.kt

```diff
@@ -95,12 +95,12 @@ extension TimestampInputInput {
 
     static func write(value: TimestampInputInput?, to writer: SmithyJSON.Writer) throws {
         guard let value else { return }
```

Comment:
Below, many codegen tests have changed to expect an explicit type on timestamp format.